rholang/src/main/scala/coop/rchain/rholang/interpreter/Substitute.scala

```diff
@@ -54,114 +56,127 @@ object Substitute {
         }
       }
 
-    ParSortMatcher.sortMatch(
-      Par(
-        for { s <- term.sends } yield substitute(s),
-        for { r <- term.receives } yield substitute(r),
-        for { e <- term.evals } yield substitute(e),
-        for { n <- term.news } yield substitute(n),
-        Nil,
-        for { m <- term.matches } yield substitute(m),
-        term.ids,
-        term.freeCount,
-        term.locallyFree.from(env.level).map(x => x - env.level)
-      ) ++ subExp(term.exprs.toList)
-    ).term.get
+    ParSortMatcher
+      .sortMatch(
+        Par(
+          for { s <- term.sends } yield substitute(s),
+          for { r <- term.receives } yield substitute(r),
+          for { e <- term.evals } yield substitute(e),
+          for { n <- term.news } yield substitute(n),
+          Nil,
```

Comment:
maybe worth naming the parameter here so I don't have to inspect `Par` to see what that `Nil` is

instead of 
```
(...)
for { n <- term.news } yield substitute(n),
Nil,
(...)
```
use
```
(...)
for { n <- term.news } yield substitute(n),
nameOfTheParameter = Nil,
(...)
```

Maybe even do it for all parameters since `Par` constructor is so huge?

Reply:
Same as above.